app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/Model/DataSet.kt

```diff
@@ -0,0 +1,40 @@
+package com.github.HumanLearning2021.HumanLearningApp.Model
+
+
+/**
+ * A class representing a data set of images
+ *
+ * @param name the name of the data set
+ * @param admin the admin who created the data set
+ * @param categories the set of categories of images present in the dataset
+ * @param comVoorLevel the ComVoor level (1-5) describing the difficulty of the data set
+ */
+abstract class DataSet(
+    val name: String,
+    val admin: Admin,
```

Comment:
Maybe make it a set of admins to allow for collaborative datasets?

Reply:
this administrator represents the creator of the dataset, right? If so, I would say there is only one creator. But if another administrator modifies the dataset, he would also be "creator"? And in this case the administrator set makes sense to me.